src/classes/RD2_Constants.cls

```diff
@@ -37,29 +37,30 @@
 public inherited sharing class RD2_Constants {
 
     /**
-    * @description Api Values of the npe03_RecurringDonation.Installment_Frequency__c picklist
+    * @description Recurring Donation "Installment Period" picklist API values
     */
-    public static final String FREQUENCY_MONTHLY = 'Monthly';
-    public static final String FREQUENCY_WEEKLY = 'Weekly';
-    public static final String FREQUENCY_YEARLY = 'Yearly';
-    public static final String FREQUENCY_FIRST_AND_FIFTEENTH = '1st and 15th';
+    public static final String INSTALLMENT_PERIOD_YEARLY = 'Yearly';
```

Comment:
@RadSFDeveloper  Thanks for noticing this and fixing. I'm not sure why I used Frequency instead of Period.